CI/Dockerfile_0_base

```diff
@@ -6,10 +6,29 @@ SHELL ["/bin/bash", "-c"]
 
 # Update core packages
 RUN apt-get -y update; \
-    apt-get -y install autoconf clang cmake g++ gcc gfortran git libblas-dev \
-                       libhdf5-dev liblapack-dev libpython2.7-dev libtool libeigen3-dev\
-                       python-numpy python-pip python-setuptools wget; \
-    pip install cython
+    apt-get -y install autoconf \
+                       clang \
+                       cmake \
+                       g++ \
+                       gcc \
+                       gfortran \
+                       git \
+                       libblas-dev \
```

Comment:
Unrelated to this PR, but I don't think we need BLAS/LAPACK anymore...?

Reply:
I forgot about this, issuing a news PR for it !
(maybe it could be a good test for our docker building/testing PR)